sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -638,8 +640,36 @@ def __init__(
     if max_batch_size is not None:
       self._batching_kwargs['max_batch_size'] = max_batch_size
     self._large_model = large_model
+
+    # Check if the device is specified twice. If true then the device parameter
+    # of model handler is overridden.
+    self._deduplicate_device_value(device)
     _validate_constructor_args_hf_pipeline(self._task, self._model)
 
+  def _deduplicate_device_value(self, device: str):
```

Comment:
Committing to speed things up in case we can get this in before release cut

Reply:
Actually, I'm about to cut and this is still 90 minutes away from being ready to merge (that's about the runtime of the longest precommit). So this will have to wait for the next release or for a cherry-pick

Reply:
sounds good! not release blocking. previous version already has a note about how to use GPU https://github.com/apache/beam/blob/27ccdc9acfc743ed89f38fd3db277cd9fe3a96f0/sdks/python/apache_beam/ml/inference/huggingface_inference.py#L586